scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingMethodImpl.java

```diff
@@ -223,11 +226,11 @@ public int schedule() {
                 NodeSet nodeSet = getRMNodes(jobMap, neededResourcesNumber, tasksToSchedule);
```

Comment:
Is it not possible to prevent most conflicts by passing `freeResources` to  `getRMNodes` and then filtering returned nodes from `SelectionManager#doSelectNodes` by using `freeResources`?